docker/assets/run.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-/root/venv/bin/python3 -m appgate $@
+/appgate/venv/bin/python3 -m appgate $@
```

Comment:
Not related directly to this PR but we could remove one extra process by using exec here:
```suggestion
exec /appgate/venv/bin/python3 -m appgate $@
```